openid-4-verifiable-presentations-1_0.md

```diff
@@ -432,54 +432,32 @@ Note: "https://self-issued.me/v2" is a symbolic string and can be used as an `au
 
 ## Verifier Metadata Management {#client_metadata_management}
 
-The `client_id_scheme` enables deployments of this specification to use different mechanisms to obtain and validate metadata of the Verifier beyond the scope of [@!RFC6749]. The term `client_id_scheme` is used since the Verifier is acting as an OAuth 2.0 Client.
+If the Client is not pre-registered with the Wallet, the `client_id` MUST be a URI.
 
-This specification defines the following values for the `client_id_scheme` parameter, followed by the examples where applicable: 
+The Wallet SHOULD verify the provided `redirect_uri` or `response_uri` belongs to the Client.
 
-* `pre-registered`: This value represents the [@!RFC6749] default behavior, i.e., the Client Identifier needs to be known to the Wallet in advance of the Authorization Request. The Verifier metadata is obtained using [@!RFC7591] or through out-of-band mechanisms.
+If the Authorization Request Object is signed, the Wallet MUST authenticate the Request Object against the `client_id` contained in the Request Object.
 
-* `redirect_uri`: This value indicates that the Verifier's Redirect URI (or Response URI when Response Mode `direct_post` is used) is also the value of the Client Identifier. The Authorization Request MUST NOT be signed. The Verifier MAY omit the `redirect_uri` Authorization Request parameter (or `response_uri` when Response Mode `direct_post` is used). All Verifier metadata parameters MUST be passed using the `client_metadata` parameter defined in (#vp_token_request).
+If the Authorization Request is signed, the `jwks` parameter in the `client_metadata` parameter MUST NOT be used to verify the signature of the Request Object.
```

Comment:
@selfissued: the above sentence eleviates the need to repeat that a JWK in the request cannot be used to verify the signature of the request which is quite obvious.